lib/constants.js

```diff
@@ -5,5 +5,8 @@ module.exports = {
     delegate: 2500000000,
     secondsignature: 500000000,
     multisignature: 500000000
+  },
+  vendorField: {
+    length: 64
```

Comment:
I'm thinking it may make more sense to call this maxLength so it doesn't get confused with string length